docs/app-host/configuration.md

```diff
@@ -0,0 +1,89 @@
+---
+title: .NET Aspire app host configuration
+description: .NET Aspire app host configuration options
+ms.date: 11/21/2024
+ms.topic: reference
+---
+
+# App host configuration
+
+The app host project configures and starts your distributed application (<xref:Aspire.Hosting.DistributedApplication>). When a `DistributedApplication` runs it reads configuration from the app host. Configuration is loaded from environment variables that are set on the app host and <xref:Aspire.Hosting.DistributedApplicationOptions>.
+
+Configuration includes:
+
+- Settings for hosting the resource service, such as the address and authentication options.
+- Settings used to start the [Aspire dashboard](../fundamentals/dashboard/overview.md), such the dashboard's frontend and OpenTelemetry Protocol (OTLP) addresses.
+- Internal settings that Aspire uses to run the app host. These are set internally but can be accessed by integrations that extend Aspire.
+
+App host configuration is provided by the app host launch profile. The app host has a launch settings file call `launchSettings.json` which has a list of launch profiles. Each launch profile is a collection of related options which defines how you would like `dotnet` to start your application.
+
+```json
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:17134;http://localhost:15170",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21030",
+        "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22057"
+      }
+    }
+  }
+}
+```
+
+The proceeding launch settings file:
+
+- Has one launch profile called `https`.
+- Configures an .NET Aspire app host project:
+  - The `applicationUrl` property configures the dashboard launch address (`ASPNETCORE_URLS`).
+  - Env vars such as `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` and `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` are set on the app host.
+
+For more information, see [.NET Aspire and launch profiles](../fundamentals/launch-profiles.md).
+
+> [!NOTE]
+> Configuration described on this page is for .NET Aspire app host project. To configure the standalone dashboard, see [dashboard configuration](../fundamentals/dashboard/configuration.md).
+
+## Common configuration
+
+| Option | Default Value | Description |
+|--|--|--|
+| `ASPIRE_ALLOW_UNSECURED_TRANSPORT` | `false` | Allows communication with the app host without https. `ASPNETCORE_URLS` (dashboard address) and `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` (app host resource service address) must be secured with HTTPS unless true. |
+
+## Resource service
+
```

Comment:
It would be nice to have an introductory sentence here, like you've done with some of the other sections. What is the resource service and why would I want to configure it?